CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Enh #789: Remove unnecessary type casting to array in `AbstractDMLQueryBuilder::getTableUniqueColumnNames()` (@Tigrov)
 - Enh #795: Allow to use `DMLQueryBuilderInterface::batchInsert()` method with empty columns (@Tigrov)
 - Enh #794: Add message type to log context (@darkdef)
+- Enh #802: Resolve psalm issues (@Tigrov)
```

Comment:
```suggestion
- Enh #802: Minor refactoring of `SchemaCache`, `AbstractPdoCommand` and `AbstractDDLQueryBuilder` (@Tigrov)
```

"Resolve psalm issues" is not about public interface of package and too common description of changes. Suggest replace it to refactoring...